jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/client/ApiClient.kt

```diff
@@ -8,6 +8,10 @@ import org.jellyfin.sdk.model.ClientInfo
 import org.jellyfin.sdk.model.DeviceInfo
 import kotlin.reflect.KClass
 
+private const val UPDATE_DEPRECATION_MESSAGE =
+	"This property should not be set directly as it does not propagate to all usages." +
```

Comment:
usages? Doesn't seem that clear to me yet.
The idea is that the setters don't properly react to changes (e.g. reconnect 
the WebSocket later on), while the update function does, right?

Reply:
Yes, although we might use it in other places eventually so I intentionally kept it a bit vague.

Reply:
Makes sense. The "propagate to all usages" (of what) could still be improved, maybe :sweat_smile: